packages/preview/sweet-graduate-resume/0.1.0/typst.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "sweet-graduate-resume"
+version = "0.1.0"
+entrypoint = "template.typ"
+authors = ["innocentzero (Md Isfarul Haque) <https://innocent_zero.codeberg.page>"]
+license = "MIT"
+description = "A simple graduate student resume template"
+repository = "https://codeberg.org/innocent_zero/typst-resume"
+categories = ["cv"]
+disciplines = ["engineering"]
+
+[template]
+path = "."
```

Comment:
This won't work as expected: with this configuration, the whole package directory would be copied when creating a project from this template. You should move `main.typ` and the images it uses to a new sub-directory.

Reply:
Oh ok. My bad. The latest version should fix it.